samples/snippets/storage_move_file.py

```diff
@@ -20,7 +20,7 @@
 from google.cloud import storage
 
 
-def move_blob(bucket_name, blob_name, destination_bucket_name, destination_blob_name):
+def move_blob(bucket_name, blob_name, destination_bucket_name, destination_blob_name,):
```

Comment:
Just a heads-up but there this sample is being removed from the documentation in favor of instruction people to copy and then delete manually - no action required here.